homeassistant/components/tuya/aes_cbc.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+"""AES-CBC encryption and decryption for account info."""
+
+import base64 as b64
+from binascii import a2b_hex, b2a_hex
+import json
+import random
+
+from Crypto.Cipher import AES
+
+AES_ACCOUNT_KEY = "o0o0o0"
+XOR_KEY = "00oo00"
+KEY_KEY = "oo00oo"
+
+
+class AesCBC:
+    """AES helper."""
+
+    def random_16(self):
+        """Return random 16."""
+        str = ""
+        return str.join(
+            random.choice("abcdefghijklmnopqrstuvwxyz!@#$%^&*1234567890")
```

Comment:
This is not a good source of randomization when used for crypto. Please use https://docs.python.org/3/library/secrets.html